tests/fixtures.py

```diff
@@ -1,7 +1,7 @@
-from honeybadgermpc.router import SimpleRouter
-import asyncio
-import random
 from pytest import fixture
+import random
+import asyncio
+from honeybadgermpc.router import SimpleRouter
```

Comment:
any reason for these changes?

Reply:
I was debugging an issue that came about importing ntl code, must have accidentally checked in this file